docs/tutorials/frontier/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Frontier Tutorial
+---
+
+[Frontier](https://github.com/paritytech/frontier) is a set of modules to build an
+**Ethereum-compatible blockchain with Substrate**.
+
+## Before You Begin
+
+You should have completed _at least_ the following three Substrate tutorials before attempting this tutorial:
+
+- [Create Your First Substrate Chain](../../tutorials/create-your-first-substrate-chain/)
+- [Add a Pallet to Your Runtime](../../tutorials/add-a-pallet/)
+- [Build a PoE Decentralized Application](../../tutorials/build-a-dapp/)
+
+Before attempting this tutorial, you should be familiar with the concepts listed below:
+
+- Launching a Substrate chain
+- Submitting extrinsics
+- Adding, Removing, and configuring pallets in a runtime
+- Ethereum and EVM basics
+- Pallet design
+
+## Template Solution and Reference
+
+> The full tutorial is presently _not up to date!_ While it is still a good reference, it relays
+> incomplete and sometimes incorrect information based an older version of the base substrate node
+> template. It is thus _highly_ encouraged to generate your own template based of whatever commit
+> you desire from frontier itself to start working with many of the core features of Frontier
+> installed and enabled.
+>
+> The biggest challenge here is that the Frontier project does not use the same, published, Substrate
+> crates; it takes Substrate code from github. Take note of this in your `Cargo` files. You _must_
+> use the _matching_ version of dependencies for all of Substrate and Frontier in your project.
+>
+> > [Old tutorial for reference](https://github.com/substrate-developer-hub/frontier-workshop/)
```

Comment:
- Do we have a dedicated place on devhub outlining how to navigate "dependency hell" in substrate? we can point there on best practices instead of re-stating it here
-  Presently you don't "do" anything but be made aware of the state of things (as a warning), and to inform people looking for the resource that have used it previously, to find it.

Reply:
https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/issues/1088 for deps and #1086 for rm of old workshop